hera_mc/db_check.py

```diff
@@ -56,12 +56,22 @@ def is_valid_database(base, session):
     True if all declared models have corresponding tables and columns.
 
     """
+    from sqlalchemy.exc import OperationalError
```

Comment:
Let's move this to the top of the file. The only reason to do imports in functions is if they require libraries that are not required by the package.